webviz_ert/data_loader/__init__.py

```diff
@@ -195,45 +195,62 @@ def get_ensemble_userdata(self, ensemble_id: str) -> dict:
     def get_ensemble_parameters(self, ensemble_id: str) -> list:
         return self._get(url=f"ensembles/{ensemble_id}/parameters").json()
 
+    def get_record_labels(self, ensemble_id: str, name: str) -> list:
+        return self._get(url=f"ensembles/{ensemble_id}/records/{name}/labels").json()
+
     def get_experiment_priors(self, experiment_id: str) -> dict:
         return json.loads(
             self._query(GET_PRIORS, id=experiment_id)["experiment"]["priors"]
         )
 
     def get_ensemble_parameter_data(
-        self, ensemble_id: str, parameter_name: str
+        self,
+        ensemble_id: str,
+        parameter_name: str,
     ) -> pd.DataFrame:
-        resp = self._get(
-            url=f"ensembles/{ensemble_id}/records/{parameter_name}",
-            headers={"accept": "application/x-parquet"},
-        )
-        stream = io.BytesIO(resp.content)
-        df = pd.read_parquet(stream)
-        return df
+        try:
+            if "::" in parameter_name:
```

Comment:
This is what I meant by future work. It might be that I'm misunderstanding something, but is this not somewhat of a hack to avoid having to make ERT2 upload parameters as it should?

Reply:
Sure we can make ert2 upload parameters like ert3 does it, but I thought the idea was that the underlying Data structure is the same for both ert3 and ert2 it should not matter how they are uploaded, ert2 or ert3 style given that both ways are valid and supported by the storage API.